src/includes/class-health-check-htaccess-viewer.php

```diff
@@ -0,0 +1,55 @@
+<?php
+/**
+ * Class for displaying the .htaccess file for the site.
+ *
+ * @package Health Check
+ */
+
+/**
+ * Class Health_Check_Htaccess
+ */
+class Health_Check_Htaccess_Viewer {
+	/**
+	 * Callback function for the tool_tabs filter that adds content
+	 * to the tools accordion.
+	 *
+	 * @param  array $tabs Associative array of tools.
+	 *
+	 * @return array Associative array.
+	 */
+	public static function tools_tab( $tabs ) {
+		// check if it is a file.
+		if ( ! is_file( ABSPATH . '.htaccess' ) ) {
+			return $tabs;
+		}
+
+		// It does so let's get the contents
+		$htaccess_file = file_get_contents( ABSPATH . '/.htaccess' );
+
+		// Remove WordPress rules.
+		$filtered_htaccess_content = trim( preg_replace( '/\# BEGIN WordPress[\s\S]+?# END WordPress/si', '', $htaccess_file ) );
+
+		// Return since it is only core added rules.
+		if ( '' === $filtered_htaccess_content ) {
+			return $tabs;
```

Comment:
I'm wondering if we should allow the displaying of the content, even if it's only got core content, avoiding conditional displaying as often as possible for a clearer user experience?

Reply:
So, display it regardless?

I'm fine with that. We'd need to add a message stating something for it too.